source/auth/auth.rst

```diff
@@ -1150,6 +1152,9 @@ Q: Why does SCRAM sometimes SASLprep and sometimes not?
 Version History
 ===============
 
+Version 1.9.1 Changes
```

Comment:
@jmikola I bumped the version for this change to 1.9.1 given that the version in #709 was bumped to 1.9.0.